schema/annotation-source.yaml

```diff
@@ -121,13 +128,17 @@ $defs:
       conclusion:
         $ref_curie: gks.core:Coding
         description: >-
-          The conclusion drawn from the statement proposition, direction, strength, and/or 
-          confidence score.
-      direction:
+          A term that describes a final conclusion about the Statement's subject, based on the statement 
+          proposition, confidence level, and/or evidence direction and strength assessments.  
+      evidence_direction:
```

Comment:
This is the name for the VA attribute that captures whether net direction of all evidence assessed supports or disputes the target proposition, or does neither (i.e. remains inconclusive).
The name it important as it fits with the naming conventions of the larger 'Evidence Level Statement' paradigm, and pairs with the 'evidence_strength' attribute.

Reply:
similar to `evidence_strength` I think `evidence_direction` should be simply `assertion_direction`, since it is about whether or not the classifying contributor is supporting or opposing the proposition.  It is not really about the evidence directly.

Reply:
I had initially proposed to map the ambiguous 'direction' and 'strength' attributes defined in the metakb-cvc model to the VA core 'evidence direction' and 'evidence strength' properties - based on the the assumption that the Statements you want to make about a target Proposition assert the net direction of all evidence evaluated (`supports` | `opposes` | `inconclusive`), along with a term and/or score reflecting the strength of the evidence in this direction.  This also seemed consistent with how we modeled other ClinGen data (e.g. Gene Validity, Dosage Sensitivity, Clinical Actionability), which similarly framed assertions as describing state of evidence surrounding some proposition. For example, Gene Validity statements asserted things like _"Moderate evidence exists supporting the proposition that GDI1 LOF mutations are pathogenic for Non-syndromic X-linked Intellectual Disability"_).   

That said, Larry and Alex's latest comments suggest that you want to frame metakb-cvc assertions in terms of our __confidence__ that the target proposition is true or false  - as opposed to assertions about the overall __strength of evidence__ for or against this target proposition.  From a SEPIO-VA perspective, this is perfectly fine - as we consider  _strength of evidence for a proposition_ to be a key factor that goes into determining _how confident we are that the proposition is true_ -  but this not be the only factor. These are separate concepts, and the VA model can represent each. For ClinVar data, it seems we want to describe things in terms of confidence level, not evidence strength.

For the v0 VA model to support this, we simply need to broaden the 'Evidence Level Statement' paradigm it currently provides to accommodate assessments of confidence that a proposition is true or false, in addition to assessments of the overall strength of evidence for or against the proposition.  Starting from the Evidence Level Statement model we already have, we can simply change the name to 'Proposition Assessment Statement' to reflect it broader remit, and shuffle in a couple properties pertaining to confidence assessment. This is what I propose in the issue comment [here](https://github.com/ga4gh/va-spec/issues/76#issuecomment-1227749798), which provides links to proposed schema, documentation, and data example updates to accommodate this use case.   

This all probably sounds very abstract right now - but it's quite a simple change, and really think it all fits with both models. **Essentially, Statements in this approach describe a target proposition, and are able to report the curators confidence that this proposition is true or false, and /or the curator's assessment of how string the evidence supports or opposes the proposition.**  I'll have some concrete examples to review on our next call.